src/components/ProgramCollection.js

```diff
@@ -70,12 +96,12 @@ class ProgramCollection extends Component {
     page: 1,
   }, () => this.update())
 
-  handleOrderingChange = (e, { name }) => this.setState({
-    ordering: this.toggleOrdering(name),
+  handleNameOrderingChange = () => this.setState({
```

Comment:
I changed this to be a dedicated function instead of reading the name off the event target, if that's ok with you.

Reply:
Is there a particular reason? Previously, it was able to handle multiple sorting options